bin/repo

```diff
@@ -54,6 +54,7 @@ module Opts
 end
 
 module Repo
+	ALLOWED_EXTENSIONS = /\.(rb|lic|xml|ui)$/
 	VERSION  = "2.32"
```

Comment:
Nit: maybe do a version bump? (but I don't think anything actually cares about this)